pandunia-lekse.csv

```diff
@@ -1285,7 +1302,9 @@ jengoxute|||zho:武术 (wǔshù), jpn:武術 (bujutsu), vie:võ thuật, kor:무
 jengrupe||||generation (cohort)||generación (cohorte)||поколение|||||||||||||
 jeni||||raw (crude)||crudo||сырой||生的|生 (未熟)||||||||||
 jeni baxe||||native language (mother tongue)|langue maternelle|lengua materna|língua materna|родной язык||母语||||मातृभाषा|||||hejmlingvo|äidinkieli|język ojczysty (język rodzimy)
+jeni moke||||log (timber)|bûche|leño|tora|бревно||原木|丸木|||बोटा (कुन्दा)||balak|kinga (gogo)||||
```

Comment:
Log means felled tree trunk, so "mokotane" could be better.

Reply:
that sounds good to me.